.github/workflows/pr.yaml

```diff
@@ -1,22 +1,17 @@
 name: "Lint and style Checks"
 on:
   pull_request:
+  push:
 jobs:
   Clippy:
     name: Clippy
     runs-on: ubuntu-latest
     container:
-      image: docker.io/mayadata/ms-buildenv:latest
+      image: docker.io/mayadata/ms-buildenv:gila
```

Comment:
same question as the previous and also on Js checks, should we update that image too?